src/components/element-editor/prefill/prefill.tsx

```diff
@@ -41,8 +41,10 @@ export const Prefill = () => {
                 });
                 return !!validationResult ? false : true;
```

Comment:
can deem validation successful as long as `validateSync()` does not throw error

```suggestion
                return true;
```

Reply:
fixed in the last push